File: CS50x/week6/sentimental-mario-less/mario.py
# TODO
from cs50 import get_int
h = get_int("Height: ")
while h < 1 or h > 8:
    h = get_int("Height: ")

for i in range(h):
    for j in range(h):
        if i + j < h - 1:
            print(" ", end="")
        elif i + j >= h - 1:
            print("#", end="")
    print()

File: CS50x/week1/mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int h, c, r;
    do
    {
        h = get_int("Please insert the height: ");
    }
    while (h<1 || h>8);

    for (r=0; r<h ; r++)
    {
        for (c=1; c <= h; c++)
        {
            if (c< (h - r))
            {
                printf(" ");
            }
            else
                printf("#");
        }
    printf("\n");
    }
}